### PR TITLE
Remove `ConsensusData` from `Block`

### LINKED ISSUE #1738: Drop consensus data
See neo-project/neo#2296 and neo-project/neo#2304.

### PR DESCRIPTION
**Pull request by fyrchik**
Close #1738 .

### CODE REVIEW

File: pkg/rpc/server/server.go [hunk 1]
```diff
@@ -1171,6 +1171,9 @@ func (s *Server) invokeContractVerify(reqParams request.Params) (interface{}, *r
 			Signers: signers,
 			Scripts: witnesses,
 		}
+		for i := range signers {
+			fmt.Println(address.Uint160ToString(signers[i].Account))
```
Comment (roman-khimov) [inline pkg/rpc/server/server.go:1175]: ?

**Review by roman-khimov — COMMENTED** [review]
The first two commits don't belong here.

**Comment by fyrchik** [comment]
First commits are needed to correctly regenerate testchain, this wasn't catched when bug was introduced.

File: pkg/interop/convert/convert.go [hunk 2]
```diff
@@ -6,11 +6,16 @@ func ToInteger(v interface{}) int {
 	return v.(int)
 }
 
-// ToByteArray converts it's argument to a ByteArray.
-func ToByteArray(v interface{}) []byte {
+// ToBytes converts it's argument to a Buffer VM type.
+func ToBytes(v interface{}) []byte {
```
Comment (AnnaShaleva) [inline pkg/interop/convert/convert.go:10]: Then we need to update `customBuiltins`.

**Review by AnnaShaleva — CHANGES_REQUESTED** [review]

File: pkg/interop/convert/convert.go (hunk 2 above)
Comment (fyrchik) [reply pkg/interop/convert/convert.go:10]: Fixed.

Comment (fyrchik) [reply pkg/interop/convert/convert.go:10]: Oh, yeah.

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**